.github/workflows/pytest-core-nompi.yml

```diff
@@ -95,13 +96,23 @@ jobs:
           python-version: '3.7'
           os: ubuntu-latest
           arch: "gcc"
+          tag: "gcc"
           language: "openmp"
           sympy: "1.10"
 
         - name: pytest-docker-py37-icc-omp
           python-version: '3.7'
           os: ubuntu-22.04
           arch: "icc"
+          tag: "icc"
+          language: "openmp"
+          sympy: "1.11"
+
+        - name: pytest-docker-py38-icx-omp
+          python-version: '3.8'
+          os: ubuntu-22.04
+          arch: "icx"
+          tag: "icc"
```

Comment:
I would just add the `icx` tag to the docker build (i,e just push the ICC image with icx tag) to avoid those

Reply:
yes I agree we don't want an extra field like "tag" here (useless complexity)

Reply:
@mloubout ?